### PR TITLE
SDIT-2123: 🔊 Reduce oracle sampling to 5% and change logging to info level

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailService.kt [hunk 1]
```diff
@@ -44,7 +44,7 @@ class DetailService(
 
     val modified = mapCmdNotificationToDetailsDto(sqlRepository.getModified())
 
-    log.info("getModified: Found ${modified.size}, time taken ${elapsed(startTime)}s")
+    log.debug("getModified: Found ${modified.size}, time taken ${elapsed(startTime)}s")
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailService.kt:47]: change to `debug` as didn't look that interesting

File: src/main/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailService.kt [hunk 2]
```diff
@@ -26,15 +26,15 @@ class DetailService(
     log.debug("Fetching shift details for $quantumId")
     // We must pad the 'from' so that we don't miss night shift ends that start the day before our from-to range.
     val details = sqlRepository.getDetails(from.minusDays(1), to, quantumId)
-    log.debug("Found ${details.size} shift details for $quantumId")
+    log.debug("Found {} shift details for {}", details.size, quantumId)
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailService.kt:29]: correct log messages

File: src/main/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailService.kt [hunk 3]
```diff
@@ -44,7 +44,7 @@ class DetailService(
 
     val modified = mapCmdNotificationToDetailsDto(sqlRepository.getModified())
 
-    log.info("getModified: Found ${modified.size}, time taken ${elapsed(startTime)}s")
+    log.debug("getModified: Found {}, time taken {}s", modified.size, elapsed(startTime))
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailService.kt:47]: change this one to debug as couldn't really see the point of it being an `info`

**Review by steverendell — APPROVED** [review]
lgtm

**Pull request merged**